sdk/template/template/review/template.api.md

```diff
@@ -19,6 +19,7 @@ export class ConfigurationClient {
 export interface ConfigurationClientOptions extends PipelineOptions {
```

Comment:
No files under `sdk/template/template` should be udpated, can you remove all the changes to these files?